src/share/cxx/Config.hpp

```diff
@@ -40,9 +44,6 @@
 #  define HOMMEXX_AVX_VERSION 1
 # else
 #  define HOMMEXX_AVX_VERSION 0
-#  ifndef HOMMEXX_VECTOR_SIZE
```

Comment:
I might be missing something, but why was this block removed? What happens in Dimensions.hpp if `HOMMEXX_AVX_VERSION == 0` and `! defined HOMMEXX_VECTOR_SIZE`?

Reply:
I might have overzealously removed it; I'll check